config/bollmaker.yaml

```diff
@@ -16,23 +16,23 @@ backtest:
   # for testing max draw down (MDD) at 03-12
   # see here for more details
   # https://www.investopedia.com/terms/m/maximum-drawdown-mdd.asp
-  startTime: "2021-08-01"
-  endTime: "2021-08-30"
+  startTime: "2022-01-01"
+  endTime: "2022-05-12"
   sessions:
   - binance
   symbols:
-  - ETHUSDT
-  account:
+  - MATICBUSD
```

Comment:
Could you copy this config to a new one for bollmaker_matic.yaml? most people trades ETH rather than MATIC. thanks!